src/Presentation/RangeFacetHtmlBuilder.php

```diff
@@ -22,13 +22,18 @@ public function __construct(
 	 * TODO: add smoke integration test
 	 */
 	public function buildHtml( FacetConfig $config, PropertyConstraints $state ): string {
+		$currentMin = $state->getInclusiveMinimum();
+		$currentMax = $state->getInclusiveMaximum();
+
 		return $this->parser->processTemplate(
 			'RangeFacet',
 			[
 				'msg-min' => wfMessage( 'wikibase-faceted-search-facet-range-min' )->text(),
 				'msg-max' => wfMessage( 'wikibase-faceted-search-facet-range-max' )->text(),
-				'current-min' => $state->getInclusiveMinimum(),
-				'current-max' => $state->getInclusiveMaximum(),
+				'msg-apply' => wfMessage( 'wikibase-faceted-search-facet-apply' )->text(),
+				'current-min' => $currentMin,
+				'current-max' => $currentMax,
+				'disabled' => $currentMin === null || $currentMax === null
```

Comment:
We allow open-ended ranges, so the button should not be disabled if any of the values are empty.

Reply:
We could disable the button by default and only enable it once any value changes. But that would probably be frontend-only and would not require passing additional data to the template.

Reply:
In that case then I will just remove the `disabled` parameter and make it the default, and have the frontend remove the `disabled` attribute

Reply:
Looks like you want to do that in a follow up?

Reply:
> Looks like you want to do that in a follow up?

Yes that part will be implemented when we make the range facet functional on the frontend